library/aos_version.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python
```

Comment:
@rhcarvalho yes, I expect that the dependency could be invoked multiple times, is there a considerable slowdown even when the included role does not actually contain any tasks, but just includes the required module(s)?

Alternatively, since you are now requiring openshift_facts, there is potential for the module to exist there instead.